lib/spark_api/models/listing_meta_translations.rb

```diff
@@ -0,0 +1,16 @@
+module SparkApi
+  module Models
+    class ListingMetaTranslations < Base
+
+      def self.for_property_type(pt, options={})
+        results = connection.get("/flexmls/propertytypes/#{pt}/translations", options)
+        if results.length > 0
+          collect(results).first
+        else
+          new(StandardFields: {}, CustomFields: {})
```

Comment:
This might be a little different from other models. When there are no results, a blank instance will still be returned, and it will respond to the same methods as an instance that has data. It should be safe to always do `ListingMetaTranslations.for_property_type('M').StandardFields['ListPrice']`, even when no translations are returned. I'd be willing to change this though if anyone feels that it's more important that it be consistent with other models.

Reply:
I'd say that's no problem